src/rpc/server.cpp

```diff
@@ -149,6 +149,7 @@ static RPCHelpMan help()
     }
     if (strCommand == "dump_all_command_conversions") {
         // Used for testing only, undocumented
+        const_cast<JSONRPCRequest&>(jsonRequest).mode = JSONRPCRequest::GET_ARGS;
```

Comment:
This is currently safe to do, because JSONRPCRequest isn't const where it is (copy-)constructed. I don't like the const_cast myself, but the only alternative I came up with was to change every rpc method with the new signature. Any other ideas? Maybe @ryanofsky

Reply:
Thanks, done